src/components/BoardView/BoardView.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+import { amount, randomNr } from "../../helpers";
+
+import CardView from "../CardView";
+
+const backgrounds = {
+  0: "pixel-dots-bg",
+  1: "wiggle-bg",
+  2: "diagonal-stripes-bg",
+  3: "rain-bg",
+  4: "zigzag-bg",
+  5: "eyes-bg"
+};
+
+const randomBg = backgrounds[randomNr(0, 5)];
+
+const Container = styled.div`
```

Comment:
Is this a first draft? In general it would be better to have the CSS in separate files.

Reply:
I believe that the classic separation of concerns does not apply to CSS-in-JS conception and styled-components, but, according to your suggestion, I moved styles to a separate file in the component directory.

Reply:
It does apply.